_guides/dhis2-app-checklist.md

```diff
@@ -0,0 +1,37 @@
+## Starting a new app
```

Comment:
Should this use `app-platform` instead of `create-react-app` or not yet?

Reply:
It should, definitely. This isn't linked anywhere on the site yet so no way to navigate to it.